src/test/java/org/opentripplanner/netex/mapping/FlexStopsMapperTest.java

```diff
@@ -107,7 +106,7 @@ void mapAreaStop() {
   }
 
   @Test
-  void mapInvalidAreaStop() {
+  void testMapInvalidAreaStop() {
```

Comment:
Since I am the opposite, I'm curious: why do you prefer to prefix the test methods with `test`?

Reply:
this is a recommendation of Sonar Cloud.
test* is the recommended pattern.

Reply:
(and for the actual reasoning: a method name should reflect what the method does. The method does not map invalid area stops, it tests what happens when invalid area stops are mapped)

Reply:
The second is actually a good point. I had never thought about it that way.